lib/install/config.go

```diff
@@ -103,10 +103,6 @@ type Config struct {
 	Mounts map[string]string
 	// DNSOverrides contains installer node DNS overrides
 	DNSOverrides storage.DNSOverrides
-	// PodCIDR is a pod network CIDR
```

Comment:
Removed this in favor of moving the corresponding values to `ClusterConfiguration` resource.